src/app/components/inputmask/inputmask.ts

```diff
@@ -94,7 +94,7 @@ export const INPUTMASK_VALUE_ACCESSOR: any = {
             (keydown)="onInputKeydown($event)"
             (keypress)="onKeyPress($event)"
             pAutoFocus
-            [variant]="variant"
+            [attr.variant]="variant"
```

Comment:
variant is an input property of pInputText, I've accidentally accepted like that. Could you please add [variant] again?

Reply:
Sure